lib/data_update_scripts/20200218195023_index_chat_channel_memberships_to_elasticsearch.rb

```diff
@@ -1,7 +1,7 @@
 module DataUpdateScripts
   class IndexChatChannelMembershipsToElasticsearch
     def run
-      ChatChannelMembership.find_each(&:index_to_elasticsearch_inline)
+      # ChatChannelMembership.find_each(&:index_to_elasticsearch_inline)
```

Comment:
should we empty the body? After all none of this is going to work if uncommented!